packages/core/src/dispatcher.ts

```diff
@@ -367,9 +367,10 @@ export class Dispatcher {
       Metrics.count(COMMITS_STORED, 1)
       return cid
     } catch (e) {
-      this._logger.err(`Error while storing commit to IPFS: ${e}`)
+      const errorMessage = `Error in repository.ts.storeInitEvent: Error - ${e} while storing commit to ipfs: ${e.message} | recon.mode: ${this.recon.enabled}`
+      this._logger.err(errorMessage)
```

Comment:
Can we capture the backtrace here so we can see which code path the dispatcher was invoked from?

Reply:
If you log an `Error` instance directly it will log the stack trace automatically I believe